### ISSUE TITLE
VoiceOver issue: list of existing selections not read when receiving focus

### ISSUE THREAD

**Issue by xinspire** [SLMNBJ/selectize-plugin-a11y#1]
I tested it using VoiceOver on OS X Sierra 10.12.6.

When a multi-select selectize control has items already populated, when the control receives tab-focus, VoiceOver does not read out existing selection.  Is this normal behavior?

**Comment by shelagh-lewins-ucl** [comment]
This is an awesome plugin, thank you. But using ChromeVox in Chrome on Linux Mint, I am still finding this bug. If I mouse click in the populated multi-select, it reads the values. But if I tab to the control, it only reads the dropdown options, not the current selection(s). Is there any way to fix this?

**Comment by jooyoungseo** [comment]
Was there any follow-up fix regarding this issue @SLMNBJ? I am a screen-reader user and looking for a solution for `selectize.js` like badly. :)

**Comment by skerit** [comment]
I don't think it's possible to fix it, without doing a lot of trickery. The ARIA rules seem to expect the text input eventually receives the value, but in selectize it is only used for filtering. When a value is chosen, the text input is emptied, confusing screen readers
